src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -1830,6 +1833,9 @@ public static Tombstone TableValueToTombstone(JsonOperationContext context, ref
                         modifiedTicks, nonPersistentFlags, newFlags);
                 }
 
+                if (ForTestingPurposesOnly().DisableDebugAssertionForTableThrowNotOwned)
+                    table.ForTestingPurposesOnly().DisableDebugAssertionForThrowNotOwned = true;
```

Comment:
I think you need to use `_forTestingPurposes` here

Reply:
done